code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -186,11 +191,13 @@
 	name = "\improper DromedaryCo packet"
 	desc = "A packet of six imported DromedaryCo cancer sticks. A label on the packaging reads, \"Wouldn't a slow death make a change?\""
 	icon_state = "Dpacket"
+	brand = "\improper Dromedary Co. cigarette"

```

Comment:
I feel they read fine as is, its flow better with the cigarette on the two its on.